src/reportSources/legacyFflogs/eventAdapter/deduplicateAoE.ts

```diff
@@ -8,6 +8,7 @@ import {AdapterStep} from './base'
  */
 export class DeduplicateAoEStep extends AdapterStep {
 	private deduplicatedEvents: Event[] = []
+	private memo: Map<string, Event> = new Map<string, Event>()
```

Comment:
style nit: I would prefer something like eventMap or something otherwise meaningful, rather than memo.  Not blocking on this comment.